common/commands.h

```diff
@@ -55,13 +59,10 @@ namespace commands::protocols {
         META(In, address, protocols);
     };
 
-    std::ostream& operator<<(std::ostream& ss, In::Protocol::Type value);
-    std::istream& operator>>(std::istream& ss, In::Protocol::Type& value);
-
     class Return : public pack::Node
     {
     public:
-        pack::String protocol = FIELD("protocol");
+        Protocol     protocol = FIELD("protocol");
```

Comment:
Please, don't change this one, need to define Protocol class in configuration manager to use in configuration